### PR TITLE
Create Network footprint.txt

### PR DESCRIPTION
**Pull request by MiladCA**

### CODE REVIEW

File: General queries/Network footprint.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// Query 1 shows you any network communication happened from endpoints to an specific Remote IP or Remote URL
```
Comment (TomerAlpert) [inline General queries/Network footprint.txt]: Typo: to a

File: General queries/Network footprint.txt [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| where RemoteIP == "IP ADDRESS GOES HERE"
+or RemoteUrl contains "URL GOES HERE"
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+
+// Query 2 shows you any network communication that happened from endpoints through a specific file to an Remote IP or Remote URL
+//Ensure to update RemoteIP, RemoteURL and InitatingProcessFileName
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == " IP ADDRESS GOES HERE"
```
Comment (TomerAlpert) [inline General queries/Network footprint.txt]: Please add parenthesis around the or

File: General queries/Network footprint.txt [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+or RemoteUrl contains " URL GOES HERE"
+and InitiatingProcessFileName == “FILENAME.EXE”
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+
+// Query 3 allows you to find network communication from specific IP or URL across MiscEvents, AlertEvents and NetworkCommunicationEvents
```
Comment (TomerAlpert) [inline General queries/Network footprint.txt]: I suggest to remove AlertEvents - it’ll only treat he events in the other tables

Comment (TomerAlpert) [reply General queries/Network footprint.txt]: Rephrasing: to find network communication to an IP or URL in the NetworkCommunicationEvents table, as well as in MiscEvents for other events (SmartScreen, launch browser with URL, more)

File: General queries/Network footprint.txt [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+// Ensure to update RemoteIP and RemoteURL variable.
+find in (MiscEvents, AlertEvents, NetworkCommunicationEvents)
+where RemoteIP == "52.114.158.52" or RemoteUrl == "google.com"
```
Comment (TomerAlpert) [inline General queries/Network footprint.txt]: Why == google?
Maybe =~, contains or endswith

File: General queries/Network footprint.txt [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == "52.169.82.131" or RemoteUrl =~ "URL GOES HERE"
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+| join kind=inner
+(
+   FileCreationEvents
+   | where EventTime > ago(30d) 
+   | project ComputerName, EventTime, FileOriginUrl, FileOriginReferrerUrl
+) on ComputerName, EventTime
```
Comment (TomerAlpert) [inline General queries/Network footprint.txt]: This join won’t work - it’s not the same event, so not th same EventTime. I also don’t understand why there are 2 joins.
I suggest to just query for file downloads from FileCreateEvents by filtering on the FileOriginUri, FileOriginIP or FileOriginReferreUrl

File: General queries/Network footprint.txt (hunk 2 above)
Comment (MiladCA) [reply General queries/Network footprint.txt]: If I use parenthesis around the or I get a syntax error:

image

image

File: General queries/Network footprint.txt (hunk 4 above)
Comment (MiladCA) [reply General queries/Network footprint.txt]: My bad ont he IP and google will fix it. ALso gonna use =~

File: General queries/Network footprint.txt (hunk 2 above)
Comment (MiladCA) [reply General queries/Network footprint.txt]: Once this is sorted I will send a new push request. Can u help me understand what I do wrong here @TomerAlpert ?

Comment (TomerAlpert) [reply General queries/Network footprint.txt]: Should do: (Cond1 or cond2) and cond3

When you do: cond1 or cond2 and cond3
It actually runs: cond1 or (cond2 and cond3)

**Pull request closed without merging**